examples/src/main/scala/org/apache/spark/examples/snappydata/CreateColumnTable.scala

```diff
@@ -0,0 +1,202 @@
+                          C_NATIONKEY: Int, C_PHONE: String, C_ACCTBAL: Double,
+                          C_MKTSEGMENT: String, C_COMMENT: String)
+
+  override def runSnappyJob(snc: SnappyContext, jobConfig: Config): Any = {
+    val pw = new PrintWriter("CreateColumnTable.out")
+    createColumnTableUsingAPI(snc, pw)
+    createColumnTableUsingSQL(snc, pw)
+    createColumnTableInferredSchema(snc, pw)
+    pw.close()
+  }
+
+  override def isValidJob(sc: SnappyContext, config: Config): SnappyJobValidation = SnappyJobValid()
+
+  def createColumnTableUsingAPI(snc: SnappyContext, pw: PrintWriter): Unit = {
+    pw.println()
+
+    pw.println("****Create a column table using API****")
+    // create a partitioned row table using SQL
+    pw.println()
+    pw.println("Creating a column table(CUSTOMER) using API")
+
+    snc.dropTable("CUSTOMER", ifExists = true)
+
+    val schema = StructType(Array(StructField("C_CUSTKEY", IntegerType, false),
+      StructField("C_NAME", StringType, false),
+      StructField("C_ADDRESS", StringType, false),
+      StructField("C_NATIONKEY", IntegerType, false),
+      StructField("C_PHONE", StringType, false),
+      StructField("C_ACCTBAL", DecimalType(15, 2), false),
+      StructField("C_MKTSEGMENT", StringType, false),
+      StructField("C_COMMENT", StringType, false)
+    ))
+
+    // props1 map specifies the properties for the table to be created
+    val props1 = Map("PARTITION_BY" -> "C_CUSTKEY", "BUCKETS" -> "11")
+    snc.createTable("CUSTOMER", "column", schema, props1)
+
+    // insert some data in it
+    pw.println()
+    pw.println("Loading data in CUSTOMER table from a text file with delimited columns")
+    val customerData = snc.sparkContext.
+        textFile(s"quickstart/src/resources/customer.tbl")
+    val customerReadings = customerData.map(s => s.split('|')).map(s => parseCustomerRow(s))
+    val customerDF = snc.createDataFrame(customerReadings)
+    customerDF.write.insertInto("CUSTOMER")
+
+    pw.println()
+    var result = snc.sql("SELECT COUNT(*) FROM CUSTOMER").collect()
+    pw.println("Number of records in CUSTOMER table after loading data are " + result(0).get(0))
+
+    pw.println()
+    pw.println("Inserting a row using INSERT SQL")
+    snc.sql("INSERT INTO CUSTOMER VALUES(20000, 'Customer20000', " +
+        "'Chicago, IL', 1, '555-101-782', 3500, 'MKTSEGMENT', '')")
+
+    pw.println()
+    result = snc.sql("SELECT COUNT(*) FROM CUSTOMER").collect()
+    pw.println("Number of records in CUSTOMER table are " + result(0).get(0))
+
+    pw.println("****Done****")
+  }
+
+  def createColumnTableUsingSQL(snc: SnappyContext, pw: PrintWriter): Unit = {
+
+    pw.println()
+
+    pw.println("****Create a column table using SQL****")
+    // create a partitioned row table using SQL
+    pw.println()
+    pw.println("Creating a column table(CUSTOMER) using SQL")
+
+    snc.sql("DROP TABLE IF EXISTS CUSTOMER")
+
+    snc.sql("CREATE TABLE CUSTOMER ( " +
```

Comment:
Folks would be interested in the various options. We should explore (maybe just through comments) how to persist, introduce HA .. at  a minimum.